app/javascript/components/users/SignInComponent.stories.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { action } from '@storybook/addon-actions';
 
 import MockCurrentUserProvider from '../tests/MockCurrentUserProvider';
-import SignInComponent, { SignInComponentProps } from './SignInComponent';
+import SignInComponent, { } from './SignInComponent';
```

Comment:
I don't know if this is on me or not, but make sure not to have empty curly brackets on imports. It's not a good practice.

Reply:
I'll fix it